### ISSUE TITLE
Error with Mario StarCatcher

### ISSUE THREAD

**Issue by nagayev** [ruffle-rs/ruffle#1840]
Hello. I ran Mario Starcatcher 2 from [this](http://www.igames.com.ua/FlashGame.aspx?name=mario_starcatcher_2) site and saw this:
![изображение](https://user-images.githubusercontent.com/16555544/101164572-42b99000-3646-11eb-8a57-1f0cb777f0d3.png)
Ruffle version:
nightly 2020-11-29

**Comment by nagayev** [comment]
Sorry, I coudn't just copy paste Error stack: Ruffle doesn't support it.

**Comment by parkerlreed** [comment]
Your Nightly is out of date. You should just see a black screen on current Nightly. (Also you should be able to highlight the text in the error message and Ctrl C)

The error it currently throws is related to AVM2

```
AVM2 error: Property does not exist: Some(AvmString { source: Owned("stage") })
```

**Comment by nagayev** [comment]
@parkerlreed Okey, thanks.

**Comment by parkerlreed** [comment]
Interestingly it works on current master on desktop

![image](https://user-images.githubusercontent.com/841440/101198335-4f7fb900-3631-11eb-8ce2-570412fd6fc7.png)

Might just need to wait for the next nightly update (Unless the website is maybe trying to play an advertisement?)

**Comment by parkerlreed** [comment]
Seems like it's the website trying to do something out of the ordinary. You can play it in the demo with this swf

https://ruffle.rs/demo/

http://www.igames.com.ua/swf/mario_starcatcher_2.swf

**Comment by nagayev** [comment]
@parkerlreed Oh, it's interesting.
Okay. So should I close this issue?

**Comment by parkerlreed** [comment]
Might be good to leave it up for a little bit so the loading error on that site can be diagnosed.

**Comment by n0samu** [comment]
The website is no longer online (likely for *geopolitical reasons...*) but it is archived here: https://web.archive.org/web/20210327051259/http://www.igames.com.ua/FlashGame.aspx?name=mario_starcatcher_2
We can see that the website uses a loader (ima3_preloader_1.5.swf) that loads the actual game. This loader SWF uses ActionScript 3. Here is the error log from running the loader:
```
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
AVM2 error: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait
Error occured when running AVM2 frame script: VerifyError: Trait loadBytes in class SafeLoader marked as override, does not override any other trait 
```
`loadBytes` is being worked on as part of #7858, so this might start working soon!

**Comment by Lord-McSweeney** [comment]
The IMA3 preloader still doesn't work, but the original game (accessible at http://web.archive.org/web/20210327201418/http://www.igames.com.ua/swf/mario_starcatcher_2.swf) is playable!

**Comment by n0samu** [comment]
Yes. That means nothing has changed. So the issue should not be closed.

**Comment by Lord-McSweeney** [comment]
Should be fixed by #20426
